src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -736,13 +741,13 @@ std::string CascadiaSettings::_ApplyFirstRunChangesToSettingsTemplate(std::strin
 // - a non-owning pointer to the scheme.
 const ColorScheme CascadiaSettings::GetColorSchemeForProfile(const GUID profileGuid) const
 {
-    auto* profile = FindProfile(profileGuid);
-    if (!profile)
+    auto profile = FindProfile(profileGuid);
+    if (profile == nullptr)
     {
         return nullptr;
     }
-    auto schemeName = profile->GetSchemeName().has_value() ? profile->GetSchemeName().value() : L"\0";
-    auto scheme = _globals.GetColorSchemes().find(schemeName);
+    auto schemeName = !profile.ColorSchemeName().empty() ? profile.ColorSchemeName() : L"\0";

```

Comment:
```suggestion
    auto schemeName = !profile.ColorSchemeName().empty() ? profile.ColorSchemeName() : L"";
```

no need to have a string literal containing `\0` because a string literal already _ends with_ `\0`

Reply:
if what you're really looking for is an hstring, just use `hstring{}`.

Reply:
Wait. Do we even need this? Can it just be:
```suggestion
    auto schemeName = profile.ColorSchemeName();
```